GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -298,7 +298,11 @@ private void DehydrateFolders(JsonTracer tracer, GVFSEnlistment enlistment, stri
                 this.ReportErrorAndExit(tracer, $"{this.ActionName} for folders failed.");
             }
 
-            this.Mount(tracer);
+            this.Mount(
+                tracer,
+                skipVersionCheck: true);  // Dehydrating folders requires that a git status be run first, and
+                                          // running git status requires that the repo already be mounted (meaning
+                                          // we don't need to check the versions again)

```

Comment:
Why did you format it this way?  Wouldn't it make just as much sense to do it like this?
```
// Dehydrating folders requires that a git status be run first, and
// running git status requires that the repo already be mounted (meaning
// we don't need to check the versions again)
this.Mount(tracer, skipVersionCheck: true);
```

Same for the other calls to `Mount`

Reply:
Good point, I will move these comments, thanks!